Devenv: Update troubleshooting documentation (#36846)
grafana/grafana: devenv/README.md

## Troubleshooting

### Containers that read from log files fail to start (Mac OS)

> If you are running Mac OSX, containers that read from the log files (e.g. telegraf, fileabeat, promtail) sometimes fail to start. This is because default Docker for Mac does not have permission to create `grafana` folder at `/var/log` location, as it runs as the current user. To solve this issue, manually create the folder `/var/log/grafana`, then try starting the containers again.

Reviewer: ```suggestion
If you are running Mac OSX, containers that read from the log files (e.g. Telegraf, Fileabeat, Promtail) can fail to start. This is because the default Docker for Mac does not have permission to create `grafana` folder at the `/var/log` location, as it runs as the current user. To solve this issue, manually create the folder `/var/log/grafana`, then start the containers again.
```

## Troubleshooting

### Containers that read from log files fail to start (Mac OS)

> If you are running Mac OSX, containers that read from the log files (e.g. telegraf, fileabeat, promtail) sometimes fail to start. This is because default Docker for Mac does not have permission to create `grafana` folder at `var/log` location, as it runs as the current user. To solve this issue, manually create the folder `/var/log/grafana`, then try starting the containers again.

Reviewer 2: ```suggestion
If you are running Mac OSX, containers that read from the log files (e.g. telegraf, fileabeat, promtail) sometimes fail to start. This is because default Docker for Mac does not have permission to create `grafana` folder at `/var/log` location, as it runs as the current user. To solve this issue, manually create the folder `/var/log/grafana`, then try starting the containers again.
```

```

> cd /
> mkdir var/log/grafana

Reviewer 2: ```suggestion
mkdir /var/log/grafana
```
 not sure if it requires `sudo` though.

Author: It actually does. Can you add it there and I'll commit.

Reviewer 2: Done :)
